static/js/settings_user_groups.js

```diff
@@ -133,32 +133,28 @@ export function populate_user_groups() {
             if (!can_edit(data.id)) {
                 return;
             }
-            const $cancel_button = $(
-                `#user-groups #${CSS.escape(data.id)} .save-status.btn-danger`,
-            );
-            const $saved_button = $(`#user-groups #${CSS.escape(data.id)} .save-status.sea-green`);
-            const $save_instructions = $(`#user-groups #${CSS.escape(data.id)} .save-instructions`);
-
-            if (is_user_group_changed() && !$cancel_button.is(":visible")) {
-                $saved_button.fadeOut(0);
-                $cancel_button.css({display: "inline-block", opacity: "0"}).fadeTo(400, 1);
-                $save_instructions.css({display: "block", opacity: "0"}).fadeTo(400, 1);
-            } else if (!is_user_group_changed() && $cancel_button.is(":visible")) {
-                $cancel_button.fadeOut();
-                $save_instructions.fadeOut();
+            const cancel_button = $(`#user-groups #${CSS.escape(data.id)} .undo-button.btn-danger`);
+            const saved_button = $(`#user-groups #${CSS.escape(data.id)} .save-status.sea-green`);
```

Comment:
This should not be changed. This change is again removed in the last commit. Please remove all these unnecessary changes.